views/blocks/sepa_mandate.tpl

```diff
@@ -42,8 +42,8 @@
   [{/if}]
 </p>
 
-[{if $oPayment->oxpayments__wdoxidee_sepamandatecustom->value}]
-  [{$oPayment->oxpayments__wdoxidee_sepamandatecustom->value}]
+[{if $sCustom}]
```

Comment:
we don't need the `if` anymore, right, we always use the text from the database!?